crux/src/Crux/Report.hs

```diff
@@ -84,10 +86,21 @@ maybeGenerateSource opts files =
 -- | Return a list of all program locations referenced in a set of
 -- proved goals.
 provedGoalLocs :: ProvedGoals -> [ProgramLoc]
-provedGoalLocs (Branch goals1 goals2) = provedGoalLocs goals1 ++
-                                        provedGoalLocs goals2
-provedGoalLocs (ProvedGoal _ err locs _) = locs ++ [simErrorLoc err]
-provedGoalLocs (NotProvedGoal _ err _ locs _) = locs ++ [simErrorLoc err]
+provedGoalLocs = concatMap Fold.toList . provedGoalTraces
+
+-- | Return a list of all of the traces referenced in a set of proved goals.
+--
+-- This returns a sequence-of-sequences because a single 'ProvedGoals' can
+-- involve many 'Branch', which mirror the branching structure of the program
```

Comment:
```suggestion
-- involve many 'Branch'es, which mirror the branching structure of the program
```

Reply:
Ah, for some reason I always thought Haddock didn't like other text surrounding the quotes. TIL! Thanks for the review.